### DISCUSSION TITLE
New Controlling plugin: Manual bookings are not added to the pdf sheet

### DISCUSSION THREAD

**Discussion by habibhaidari1** [kimai/kimai#4355] [General]
Hello, 
we are using the new controlling plugin and found a possible bug. After adding manual bookings for a specific date, we expect the overtime or undertime to be corrected by the corresponding amount in the day view. However, these values do not change. Also in the generated PDF file these manual bookings are not included. 

I think that is a possible bug because we are maintaining it so that we have in the PDF a working time corrected by the corresponding value from the manual booking. will this possibly be adjusted in the future?

**Comment by kevinpapst** [comment]
Please post plugin topics always here: https://github.com/kevinpapst/kimai-plugins
